### PR TITLE
Introduce Generated SDK 

### PR DESCRIPTION
**Pull request by andrii-balitskyi**

### CODE REVIEW

File: .github/workflows/dependabot-auto-merge.yml
Comment (razor-x) [inline .github/workflows/dependabot-auto-merge.yml:1]: Can you use this workflow, it's slightly improved to use the seambot token https://github.com/seamapi/javascript-next/blob/main/.github/workflows/automerge.yml

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+seamapi/** linguist-generated
```
Comment (razor-x) [inline .gitattributes]:
```suggestion
seamapi/** linguist-generated

```

File: .github/workflows/generate.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        run: npm run generate
+      - name: Format code
+        run: poetry run black ./seamapi
+      - name: Commit
+        uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          commit_message: "ci: Generate code"
+          commit_user_name: ${{ secrets.GIT_USER_NAME }}
+          commit_user_email: ${{ secrets.GIT_USER_EMAIL }}
+          commit_author: ${{ secrets.GIT_USER_NAME }} <${{ secrets.GIT_USER_EMAIL }}>
```
Comment (razor-x) [inline .github/workflows/generate.yml]: Where are this missing line endings coming from?
```suggestion
          commit_author: ${{ secrets.GIT_USER_NAME }} <${{ secrets.GIT_USER_EMAIL }}>
          
```

File: .npmrc
Comment (razor-x) [inline .npmrc:1]: This file should be removed.

**Comment by razor-x** [comment]
Do we know why the tests are failing?

File: seamapi/access_codes_unmanaged.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+
+class AccessCodesUnmanaged(AbstractAccessCodesUnmanaged):
+    seam: Seam
+
+    def __init__(self, seam: Seam):
+        self.seam = seam
+
+    def convert_to_managed(
+        self,
```
Comment (razor-x) [inline seamapi/access_codes_unmanaged.py:16]: Let's enforce keyword arguments for all methods: https://thepythoncodingbook.com/2022/12/11/positional-only-and-keyword-only-arguments-in-python/
```suggestion
        self, /,
```

File: package.json [hunk 4]
```diff
@@ -24,11 +23,7 @@
   },
   "homepage": "https://github.com/seamapi/python#readme",
```
Comment (razor-x) [inline package.json]:
```suggestion
  "private": true,
  "homepage": "https://github.com/seamapi/python",
```

**Comment by andrii-balitskyi** [comment]
> Do we know why the tests are failing?

@razor-x we do, there are no /webhooks endpoints on fake-seam-connect. I'll update the fake.

File: .github/workflows/dependabot-auto-merge.yml
Comment (andrii-balitskyi) [reply .github/workflows/dependabot-auto-merge.yml:1]: https://github.com/seamapi/python/pull/142/commits/e65ff453fbd68c0f6fc2b98e0f7bcb3f5b31c163

File: .gitattributes (hunk 1 above)
Comment (andrii-balitskyi) [reply .gitattributes]: https://github.com/seamapi/python/pull/142/commits/56fe1277f8c2896edfcfd03f5f28a4956178ad14

File: .github/workflows/generate.yml (hunk 2 above)
Comment (andrii-balitskyi) [reply .github/workflows/generate.yml]: https://github.com/seamapi/python/pull/142/commits/56fe1277f8c2896edfcfd03f5f28a4956178ad14

File: package.json (hunk 4 above)
Comment (andrii-balitskyi) [reply package.json]: https://github.com/seamapi/python/pull/142/commits/e1050c800b9332db0cd98cd71e2a56569c420c4c

File: seamapi/access_codes_unmanaged.py (hunk 3 above)
Comment (andrii-balitskyi) [reply seamapi/access_codes_unmanaged.py:16]: @razor-x what about Seam client constructor? Should we enforce keyword-only args there as well?

Comment (razor-x) [reply seamapi/access_codes_unmanaged.py:16]: Yes, but let's allow api key to be used in these ways, all other args are keyword-only

```py
seam = Seam() # reads SEAM_API_KEY from env
seam = Seam('api-key') 
seam = Seam(api_key='api-key') 
```

File: .npmrc
Comment (razor-x) [inline .npmrc:1]: Remove this file

File: seamapi/access_codes_unmanaged.py (hunk 3 above)
Comment (andrii-balitskyi) [reply seamapi/access_codes_unmanaged.py:16]: https://github.com/seamapi/python/pull/142/commits/32fdb8fea1efe46084aeadb4520a5fc0ca4e81d2

File: .npmrc
Comment (andrii-balitskyi) [reply .npmrc:1]: https://github.com/seamapi/python/pull/142/commits/89156fdcde874ff153a5deb7b198d8b7ad722678

**Comment by razor-x** [comment]
Moved to https://github.com/seamapi/python-next/pull/6

**Pull request closed without merging**